config/routes.rb

```diff
@@ -102,7 +102,7 @@
     resources :bundles do
       get :manage, on: :member
       resources :bundle_products, controller: "bundle_products", except: [:show]
-      resources :file_uploads, path: "files", only: [:new, :create, :destroy]
+      resources :file_uploads, path: "files", only: [:index, :create, :destroy]
```

Comment:
Correct, it's not being used. The upload form is on same view as the index list.